easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20180622-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'dcm2niix'
+version = '1.0.20180622'
+
+homepage = 'https://github.com/rordenlab/dcm2niix'
+description = """dcm2niix is a designed to convert neuroimaging data from the DICOM format to the NIfTI format."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/rordenlab/dcm2niix/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e9f79509f44aac82c9663381f8f4bfb18a9a3c3eb112d418c92629a871bbb13c']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```

Comment:
@crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead